src/molecules/ProfileDropdown.tsx

```diff
@@ -25,6 +26,13 @@ const Button = styled.div`
 `;
 
 function ProfileDropdown() {
+	if (signInStatus) {
+		DropdownItems[2].title = 'Sign Out';
+		DropdownItems[2].path = '/signout';
+	} else {
+		DropdownItems[2].title = 'Sign In';
+		DropdownItems[2].path = '/signin';
+	}
```

Comment:
signInStatus = needs to be passed down from the gradebook as a prop. Importing the current way it won't receive updates